src/app/dev/DevToys.Tools/Tools/Text/ListCompare/ListCompareCommandLineTool.cs

```diff
@@ -0,0 +1,102 @@
+﻿using DevToys.Tools.Helpers;
+using DevToys.Tools.Models;
+using Microsoft.Extensions.Logging;
+using OneOf;
+
+namespace DevToys.Tools.Tools.Text.ListCompare;
+
+[Export(typeof(ICommandLineTool))]
+[Name("ListCompare")]
+[CommandName(
+    Name = "listcompare",
+    Alias = "lc",
+    ResourceManagerBaseName = "DevToys.Tools.Tools.Text.ListCompare.ListCompare",
+    DescriptionResourceName = nameof(ListCompare.Description))]
+internal sealed class ListCompareCommandLineTool : ICommandLineTool
+{
+#pragma warning disable IDE0044 // Add readonly modifier
+    [Import]
+    private IFileStorage _fileStorage = null!;
+#pragma warning restore IDE0044 // Add readonly modifier
+
+    [CommandLineOption(
+        Name = "filea",
+        Alias = "a",
+        IsRequired = true,
+        DescriptionResourceName = nameof(ListCompare.PathListA))]
+    private OneOf<FileInfo, string>? FileA { get; set; }
```

Comment:
Does it make sense to use `OneOf` here? How would a user use this parameter with a string instead of a file, considering that (and correct me if I'm wrong) I imagine that in most cases, user will want to compare text that has multiple lines. It's likely not easy to do in command line without using a file. Am I missing something?

Reply:
That's right